### PR TITLE
Fix bad color RGB/BGR

### PR DESCRIPTION
**Pull request by anlhord**
Fix virtualbox RDP is displayed in bad color.

### CODE REVIEW

**Comment by anlhord** [comment]
It builds. I've just tested in on wayland and x11/weston.

File: client/Wayland/wlfreerdp.c [hunk 1]
```diff
@@ -74,10 +74,19 @@ static BOOL wl_end_paint(rdpContext* context)
 	display = context_w->display;
 	window = context_w->window;
 
-	for (i = 0; i < h; i++)
+	for (i = 0; i < h; i++) {
```
Comment (bmiklautz) [inline client/Wayland/wlfreerdp.c]: Please note that this coding style isn't common for FreeRDP. The bracket should be on the next line.

**Comment by bmiklautz** [comment]
@freerdp-bot test

**Comment by bmiklautz** [comment]
@anlhord did you test your changes against any other server than VirtualBox RDP?

**Comment by hardening** [comment]
I confirm bug and correction
Just curious: don't we have function to perform that kind of operation in FreeRDP ?

**Comment by bmiklautz** [comment]
@hardening we do. I didn't look at the code yet but I would assume that gdi is initialized with the wrong color format?

**Comment by hardening** [comment]
@anlhord Have you tried initializing gdi without CLRCONV_INVERT ?

**Comment by hardening** [comment]
@anlhord I've just done a test and removing CLRCONV_INVERT produce the expected output

**Comment by hardening** [comment]
I'm closing this PR as I think your proposal is not the "right way to do it"(TM).

**Comment by anlhord** [comment]
Can I ask:
1. how to fix this
2. how to also implement keyboard/mouse/touch input (preferably without threads since thats hard for me to debug)

**Comment by hardening** [comment]
@anlhord except if I missed something, the color inversion bug is fixed on master. The input bug is something else.

**Pull request closed without merging**